projects/common/src/constants/serverInfo.ts

```diff
@@ -1 +1,14 @@
 export const apiUrl = 'https://api.sasil.app' as const;
+
+export const googleClientId = {
+  web: '1039863031545-qeai77l9fsp4lpuvgj99q5hmlspeooij.apps.googleusercontent.com',
+  expo: '1039863031545-qeai77l9fsp4lpuvgj99q5hmlspeooij.apps.googleusercontent.com',
+  ios: '1039863031545-4in5ljvaephst13ridnjtqm32ododsho.apps.googleusercontent.com',
+  android:
+    '429457146313-mq693iuckrp5spudm138g1rnntgnjjt4.apps.googleusercontent.com',
+};
+export const appleClientId = 'com.sasil.sasilservice';
+export const appleRedirectURI =
+  'https://sasil-front-git-apple-login-test-sasil.vercel.app/login';
+export const kakaoClientId = '2a129108cde2d21e8b1e24fb86e350e7';
+export const kakaoRedirectURI = 'https://sasil.app/login';
```

Comment:
🍎  만약 로그인 페이지 주소를 `/auth`로 할거면 여기 수정해줘야해! 나는 `/login`으로 생각하고 코드 짰던거여서..
그리고 애플 `redirectURI`도 `https://sasil.app/auth`로 변경해줘야 하고! (기존 값은 테스트 주소고, QA할때 이 부분만 직접 변경해줘서 테스트하는게 맞는것 같아)

아니면 로그인 페이지 주소를 `/login`으로 바꾸는 것도 방법이고,, 두 방법 중 하나 선택해서 알려주세여!
```suggestion
export const kakaoRedirectURI = 'https://sasil.app/auth';
```